github/token

```diff
@@ -1,6 +1,21 @@
-#!/bin/bash
+#!/usr/bin/env ruby
+# frozen_string_literal: true
 
-JSON=$(curl -X POST -H "Authorization: Bearer $(./github/jwt)" -H "Accept: application/vnd.github.machine-man-preview+json" "https://api.github.com/installations/$INSTALLATION/access_tokens")
-TOKEN=$(jq -r '.token' <<< $JSON)
+require "json"
+require "net/http"
+require "uri"
 
-echo $TOKEN
+jwt = File.expand_path("jwt", __dir__)
+
+uri = URI("https://api.github.com/installations/#{ENV["INSTALLATION"]}/access_tokens")
+req = Net::HTTP::Post.new(uri)
+req["Authorization"] = "Bearer #{`#{jwt}`}"
+req["Accept"] = "application/vnd.github.machine-man-preview+json"
+
+http = Net::HTTP.new(uri.hostname, uri.port)
+http.use_ssl = (uri.scheme == "https")
+res = http.request(req)
```

Comment:
I would highly recommend using [octokit's built-in method for this](https://github.com/octokit/octokit.rb/blob/3740b9b6fd3e06f3ead5392a9d3ef618f8b22003/lib/octokit/client/apps.rb#L56-L68) if you're going to be using Ruby.